### PR TITLE
Answer:39

### PR DESCRIPTION
**Pull request by GRDtechlab**
**Anwer:39**
Committing for angular challenge **Anwer:39**
Injection Token to provide different timer value.

## Checklist for challenge submission

- [ ] Start your PR message with Answer:${challenge_number}

### CODE REVIEW

File: apps/angular/injection-token/src/app/app.component.ts [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import { Component } from '@angular/core';
 import { RouterLink, RouterOutlet } from '@angular/router';
+import { DEFAULT_TIMER } from './data';
 
 @Component({
   standalone: true,
   imports: [RouterOutlet, RouterLink],
+  providers: [{ provide: DEFAULT_TIMER, useValue: 1000 }],
```
Comment (tomalaforge) [inline apps/angular/injection-token/src/app/app.component.ts:8]: You can set this directly inside the injectionToken.

**Comment by tomalaforge** [comment]
Thanks for submitting your work. There is just one improvement 👍

**Comment by GRDtechlab** [comment]
Created new committed code to provide token value directly inside injection file rather than providing value in app-component.

**Comment by tomalaforge** [comment]
This is perfect. Thanks for your solution

**Pull request closed without merging**